common/test/acceptance/tests/studio/test_studio_course.py

```diff
@@ -0,0 +1,432 @@
+from ...pages.studio.course_info import CourseUpdatesPage
+from ...pages.studio.edit_tabs import PagesPage
+from ...pages.studio.users import CourseTeamPage
+from ...pages.studio.settings import SettingsPage
+from ...pages.studio.settings_advanced import AdvancedSettingsPage
+from ...pages.studio.settings_graders import GradingPage
+from ...pages.studio.textbooks import TextbooksPage
+from ...pages.studio.index import DashboardPage
+from ...pages.studio.overview import CourseOutlinePage
+
+from ...pages.studio.import_export import ExportCoursePage, ImportCoursePage
+
+
+class CoursePagesTest(StudioCourseTest):
+    """
+    Tests that verify the pages in Studio that you can get to when logged
+    in and have a course.
+    """
+
+    COURSE_ID_SEPARATOR = "."
+
+    def setUp(self, is_staff=False):
+        """
+        Install a course with no content using a fixture.
+        """
+        super(CoursePagesTest, self).setUp()
+
+        self.pages = [
+            clz(self.browser, self.course_info['org'], self.course_info['number'], self.course_info['run'])
+            for clz in [
+                AssetIndexPage, ChecklistsPage, ImportCoursePage, CourseUpdatesPage,
+                PagesPage, ExportCoursePage, CourseTeamPage, CourseOutlinePage, SettingsPage,
+                AdvancedSettingsPage, GradingPage, TextbooksPage
+            ]
+        ]
+
+    def test_page_redirect(self):
+        """
+        /course/ is the base URL for all courses, but by itself, it should
+        redirect to /home/.
+        """
+        self.dashboard_page = DashboardPage(self.browser)  # pylint: disable=attribute-defined-outside-init
+        self.dashboard_page.visit()
+        self.assertEqual(self.browser.current_url.strip('/').rsplit('/')[-1], 'home')
+
+    @skip('Intermittently failing with Page not found error for Assets. TE-418')
+    def test_page_existence(self):
+        """
+        Make sure that all these pages are accessible once you have a course.
+        Rather than fire up the browser just to check each url,
+        do them all sequentially in this testcase.
+        """
+
+        # In the real workflow you will be at the dashboard page
+        # after you log in. This test was intermittently failing on the
+        # first (asset) page load with a 404.
+        # Not exactly sure why, so adding in a visit
+        # to the dashboard page here to replicate the usual flow.
+        self.dashboard_page = DashboardPage(self.browser)  # pylint:disable=attribute-defined-outside-init
+        self.dashboard_page.visit()
+
+        # Verify that each page is available
+        for page in self.pages:
+            page.visit()
+
+
+@attr('shard_2')
+class CourseTeamPageTest(StudioCourseTest):
+    """ As a course author, I want to be able to add others to my team """
+    def _make_user(self, username):
+        """ Registers user and returns user representation dictionary as expected by `log_in` function """
+        user = {
```

Comment:
@Kelketek to register a user but do not log in. This is confusing, so I removed `no_login` parameter from `log_in` function and explicitly used `AutoAuthPage` instead